### PR TITLE
Remove RC and ready the actual release

### PR DESCRIPTION
**Pull request by mattdurham**
Update v0.26.0-rc.0 to v0.26.0

### CODE REVIEW

File: tools/release-note.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 This is release `${RELEASE_TAG}` of the Grafana Agent.
 
 ### Upgrading
-Read the [upgrade guide](https://github.com/grafana/agent/blob/${RELEASE_TAG}/docs/upgrade-guide/_index.md) for specific instructions on upgrading from older versions.
+Read the [upgrade guide](https://github.com/grafana/agent/blob/${RELEASE_TAG}/docs/sources/upgrade-guide/_index.md) for specific instructions on upgrading from older versions.
```
Comment (rfratto) [inline tools/release-note.md:4]: Can we change this to https://grafana.com/docs/agent/latest/upgrade-guide/?

Comment (rfratto) [reply tools/release-note.md:4]: `https://grafana.com/docs/agent/v0.26/upgrade-guide`? We'll need some magic to pull out the major and minor version, but I think we should point people at the rendered documentation.

Comment (mattdurham) [reply tools/release-note.md]: Added code to pull out the major/minor version for release docs

File: .drone/drone.yml [hunk 2]
```diff
@@ -243,6 +243,7 @@ steps:
       - go install github.com/tcnksm/ghr@latest
       - go install github.com/mitchellh/gox@latest
       - export PATH="$(go env GOPATH)/bin:$PATH"
+      - export RELEASE_DOC_TAG=`echo ${DRONE_TAG} | awk -F "\." '{print $1"."$2}'`
```
Comment (rfratto) [inline .drone/drone.yml:246]: This should probably be moved to the release bash script so it still works directly from make

File: Makefile [hunk 3]
```diff
@@ -503,6 +503,7 @@ clean-dist:
 .PHONY: clean
 
 publish: dist
+	export RELEASE_DOC_TAG=`echo ${RELEASE_TAG} | awk -F "\." '{print $1"."$2}'`
```
Comment (rfratto) [inline Makefile:506]: I don't think this works in make, does it?

**Review by marctc — APPROVED** [review]

File: Makefile (hunk 3 above)
Comment (mattdurham) [reply Makefile]: You are correct, fixing

File: Makefile [hunk 4]
```diff
@@ -18,6 +18,8 @@ RELEASE_TAG = $(IMAGE_TAG)
 else
 IMAGE_TAG ?= $(RELEASE_TAG)
 
+RELEASE_DOC_TAG = `echo ${RELEASE_TAG} | awk -F "\." '{print $$1"."$$2}'`
```
Comment (rlankfo) [inline Makefile:21]:
```suggestion
RELEASE_DOC_TAG = `echo ${RELEASE_TAG} | awk -F '.' '{print $$1"."$$2}'`
```

**Review by rlankfo — APPROVED** [review]

File: Makefile (hunk 4 above)
Comment (mattdurham) [reply Makefile:21]: Wierd, I didn't see a warning but maybe differences in versions? Either works for me (OSX) but will change

Comment (rlankfo) [reply Makefile]: ahh looks like BSD awk doesn't print the warning, but GNU awk does which is what the build env uses:
```
➜  ~ echo 'v0.26.0' | /usr/bin/awk -F"\." '{print $2}'
26
➜  ~ echo 'v0.26.0' | /usr/local/bin/gawk -F"\." '{print $2}'
gawk: warning: escape sequence `\.' treated as plain `.'
26
```

Comment (rlankfo) [reply Makefile]:
```
➜  ~ /usr/bin/awk --version
awk version 20200816
➜  ~ /usr/local/bin/gawk --version
GNU Awk 5.1.1, API: 3.1 (GNU MPFR 4.1.0, GNU MP 6.2.1)
Copyright (C) 1989, 1991-2021 Free Software Foundation.
```

**Pull request merged**